tests/integration/http-verbs-test.js

```diff
@@ -99,6 +103,20 @@ test('mirage responds to patch', function(assert) {
   });
 });
 
+test('mirage responds to resource', function(assert) {
+  assert.expect(0);
+  let done = assert.async();
+
+  this.server.resource('/contacts');
```

Comment:
I was actually thinking if it's better to be more expressive (`this.resource('contacts')`) or keep the symmetry with other shorthands (`this.post(/contacts)`). For me both would be fine here,  no problem to change the api if you prefer `this.resource('contacts')` syntax.